core/src/main/java/eu/mhutti1/utils/storage/StorageDeviceUtils.kt

```diff
@@ -36,6 +36,14 @@ object StorageDeviceUtils {
       add(environmentDevices(context))
       addAll(externalMountPointDevices())
       addAll(externalFilesDirsDevices(context, false))
```

Comment:
@MohitMaliFtechiz Then we shouldn't need this anymore? right?

Reply:
@gouri-panda We need this because it scans for the device's main directory when we pass `false` into it.

```
private fun generalisePath(path: String, writable: Boolean) =
    if (writable) path
    else path.substringBefore("/Android/data/")
```